### PR TITLE
A Good Descriptive Title

### PR DESCRIPTION
**Pull request by opus49**
## Description

This was a feature added during the JEDI Academy.

## Definition of Done

One step past medium-well.

### Issue(s) addressed

N/A

## Dependencies

N/A

## Impact

Things will never quite be the same again.

### CODE REVIEW

**Comment by GlennCreighton** [comment]
I think this is fantastic, it's probably the BEST ADDITION TO JEDI WE HAVE SEEN YET!  Congratulations!!!

**Comment by twsearle** [comment]
We should merge this one, if only for the reference to cooking steak.

**Review by twsearle — APPROVED** [review]

**Comment by andrewlorenc** [comment]
I prefer medium-rare, but each to his own, so I approve.

File: test/testinput/qc_practical_boundscheck.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    minvalue: 14.0
+    maxvalue: 19.0
+#  Compare variables with minvalue/maxvalue
+#  variable1@ObsValue = 10, 11, 12, 13, 14, 15, 16, 17, 18, 19
+#  variable2@ObsValue = 10, 12, 14, 16, 18, 20, 22, 24, 26, 28
+#  variable3@ObsValue = 25, 24, 23, 22, 21, 20, 19, 18, 17, 16
+#  number of data points passing the filter
+  passedBenchmark: 13
+
+observations:
```
Comment (GlennCreighton) [inline test/testinput/qc_practical_boundscheck.yaml:27]: i think having this in here was causing it only to run the second test

**Pull request closed without merging**